framework/src/test/java/org/tron/common/config/args/ArgsTest.java

```diff
@@ -45,4 +47,18 @@ public void testConfig() {
     Assert.assertEquals(rateLimiter.getHttpMap().size(), 1);
     Assert.assertEquals(rateLimiter.getRpcMap().size(), 0);
   }
+
+  @Test
+  public void testHelpMessage() {
```

Comment:
Please submit the tests to cover  this PR, if it is simply to increase coverage, it is recommended to submit a new PR.

Reply:
i will split this pr later.